src/components/inspectors/process.js

```diff
@@ -16,7 +16,7 @@ export default {
             initiallyOpen: true,
             label: 'Configuration',
             icon: 'cog',
-            name: 'configuration',
```

Comment:
This name isn't displayed visually anywhere, but is required to be the same to group all accordions so they act in tandem (i.e. as one opens, the other closes).